src/inbox.rs

```diff
@@ -81,41 +83,45 @@ impl<A> Chan<A> {
         let mut inner = self.chan.lock().unwrap();
 
         let result = match message {
-            SentMessage::ToAllActors(m) if !self.is_full(inner.broadcast_tail) => {
+            SentMessage::ToAllActors(m) => {
+                if self.is_full(inner.broadcast_tail) {
+                    // on_shutdown is only notified with inner locked, and it's locked here, so no race
```

Comment:
Comment is a bit odd here?

Reply:
It is not my comment, it was already there, previously on line 89. I don't fully understand it to be honest so I just left it with the same code as it was before.

Reply:
It appeared here: https://github.com/Restioson/xtra/pull/113/files#diff-5a063c294075054486f1cce09b584344bdea5f8ddec0f4551159f038318908d3R46

Reply:
Huh. I think it can be removed now. Probably a copy-paste error on my part